src/common/components/serviceNotifications/ServiceNotifications.test.tsx

```diff
@@ -39,13 +39,8 @@ describe('ServiceNotifications', () => {
     'renders %s notification and handles close action',
     async (bannerType) => {
       server.use(
-        rest.get('/api/banners', async (_, res, ctx) => {
-          return res(
-            ctx.status(200),
-            ctx.json({
-              [bannerType]: BANNERS[bannerType],
-            }),
-          );
+        http.get('/api/banners', async () => {
+          return HttpResponse.json({ [bannerType]: BANNERS[bannerType] }, { status: 200 });
```

Comment:
In case of returning 200, it can be omitted:
```
    server.use(
      http.get('/api/banners', async () => {
        return HttpResponse.json(BANNERS);
      }),
    );
```

Reply:
This made the code a lot cleaner in some places. Thank you for the suggestion.